cmd/preview/deploy.go

```diff
@@ -87,7 +87,7 @@ func Deploy(ctx context.Context) *cobra.Command {
 	}
 	cmd.Flags().StringVarP(&opts.branch, "branch", "b", "", "the branch to deploy (defaults to the current branch)")
 	cmd.Flags().StringVarP(&opts.repository, "repository", "r", "", "the repository to deploy (defaults to the current repository)")
-	cmd.Flags().StringVarP(&opts.scope, "scope", "s", "personal", "the scope of preview environment to create. Accepted values are ['personal', 'global']")
+	cmd.Flags().StringVarP(&opts.scope, "scope", "s", "global", "the scope of preview environment to create. Accepted values are ['personal', 'global']. Default is 'global'")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&opts.scope, "scope", "s", "global", "the scope of preview environment to create. Accepted values are ['personal', 'global']")
```

I would remove this because if you run `okteto preview deploy --help` it will repeat the default twice, it looks like Cobra is already appending the default:

`-s, --scope string        the scope of preview environment to create. Accepted values are ['personal', 'global']. Default is 'global' (default "global")`